config/environments/development.rb

```diff
@@ -10,10 +10,22 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local = true
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+      address: "smtp.mandrillapp.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.